src/components/sidebar/LinkView.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { inject, observer } from 'mobx-react';
+import { SidebarStore } from '../../stores/sidebarStore';
+import * as s from './linkView.scss';
+
+interface IViewViewProps {
+    sidebarStore?: SidebarStore;
+}
+
+@inject('sidebarStore')
+@observer
+class LinkView extends React.Component
+<IViewViewProps> {
```

Comment:
Personally I prefer lines 12 and 13 to be combined, but we haven't agreed on any specifics..